Modules/MFT/src/QcMFTAsyncTask.cxx

```diff
@@ -131,10 +131,12 @@ void QcMFTAsyncTask::initialize(o2::framework::InitContext& /*ctx*/)
     mTrackXYNCls[nHisto] = std::make_unique<TH2F>(Form("tracks/mMFTTrackXY_%d_MinClusters", minNClusters), Form("Track Position (NCls >= %d); x; y", minNClusters), 320, -16, 16, 320, -16, 16);
     mTrackXYNCls[nHisto]->SetOption("COLZ");
```

Comment:
I believe this line should be removed since the option is now set in the new line 134.